data/eos-demo-video.desktop.in.in

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+_Name=Demo Video
+_Comment=Demonstration Video shown for In-Store units
```

Comment:
No need to translate these, or even add a comment at all. Strings from autostart desktop files are never displayed to the user.